src/elements/dot-state-icon/dot-state-icon.tsx

```diff
@@ -27,24 +28,27 @@ export class DotContentletStateIcon {
                 }}
             >
                 <span>
-                    <div class={state} id="icon"></div>
+                    <div class={state} id="icon" />
                     <dot-tooltip content={name} for="icon" />
                 </span>
             </Host>
         );
     }
 
-    private getType({ live, working, deleted, hasLiveVersion }: DotContentletItem): string {
-        if (deleted === 'true') {
+    private getType({ live, working, deleted, hasLiveVersion }: DotContentState): string {
+        if (deleted === 'true' || deleted == true) {
             return 'archived'; // crossed
         }
 
-        if (live === 'true') {
-            if (hasLiveVersion === 'true' && working === 'true') {
+        if (live === 'true' || live == true) {
```

Comment:
A lots of "ors" here.

What about 

```typescript
function isTrue(val: string | boolean): boolean {
    return val === 'true' || val == true
}

```
```suggestion
        if (isTrue(live)) {
```

Reply:
or you could **do cast to string** at the beginning of the method, and leave everything as is!

Reply:
`const isTrue = (val: string | bool) => JSON.parse(val)`